nicegui/testing/__init__.py

```diff
@@ -1,4 +1,15 @@
-from .screen import Screen
+try:
+    from .screen import Screen
+except ImportError:
+    # we simply define Screen as None if selenium is not installed
+    # this allows simpler dependency management when only using the "User" fixture
+    class Screen:  # type: ignore
+        def __init__(self, *args, **kwargs):
+            raise ImportError('Selenium dependencies are missing. Please install selenium to use the Screen class.')
```

Comment:
Are we sure Selenium is missing?
Maybe we can re-raise the original import error?

Reply:
Good idea. I added it in [5d6a236](https://github.com/zauberzeug/nicegui/pull/3585/commits/5d6a2365676e049b079fa852cfd1f2c559895563).